input/fsh/operation-profiles/versionable-operation-profile.fsh

```diff
@@ -0,0 +1,39 @@
+Profile: VersionableBindableOperation
```

Comment:
This becomes CanonicalVersionBindable operation, also create a new ArtifactVersionBindable operation that defines artifactVersion, checkArtifactVersion, and forceArtifactVersion parameters